static/src/javascripts/projects/common/modules/article/space-filler.js

```diff
@@ -24,6 +24,7 @@ class SpaceFiller {
 			fastdom.mutate(() => writer(paragraphs));
 
 		const onNoSpacesFound = (ex) => {
+			console.log(ex);
```

Comment:
Not important: do we want to check in this log?

Reply:
It would probably be better to use something like [reportError](https://github.com/guardian/frontend/blob/c493d08322276c1c5eb66b3ccbe8183ca210641e/static/src/javascripts/lib/report-error.js#L11) but I reckon logging to the console is better than nothing.